release_manifests/24.10-LTS25.01.yaml

```
@@ -1,3 +1,4 @@
+ga_version: TBD
```

```suggestion
release_version: TBD
```

Maybe change ga_version name since this is a bit confusing under LTS manifest